service/nodeconfig/v1/key/key.go

```diff
@@ -0,0 +1,20 @@
+package key
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/microerror"
+)
+
+func ToCustomObject(v interface{}) (v1alpha1.NodeConfig, error) {
```

Comment:
I'm leaning towards naming this ToNodeConfig, etc. WDYT?

Reply:
This means more renaming work when you copy it around for other purposes. I like it is generic. No really strong feelings though.